tpot/base.py

```diff
@@ -641,6 +641,47 @@ def export(self, output_file_name):
         with open(output_file_name, 'w') as output_file:
             output_file.write(export_pipeline(self._optimized_pipeline, self.operators, self._pset))
 
+    def _contains_nan(self, features):
+        """Determine if a feature set contains any missing values.
+
+        Parameters
+        ----------
+        features: array-like {n_samples, n_features}
+
+        Returns
+        -------
+        bool, True if NaN is present, False otherwise.
+        """
+        return np.isnan(np.sum(features))
```

Comment:
Are you *sure* that `np.isnan(np.sum(features))` is faster than `np.any(np.isnan(features))`?